### DISCUSSION TITLE
RadioSet: missing method action_toggle_button

### DISCUSSION THREAD

**Discussion by UlrichGoebel** [Textualize/textual#4569] [Q&A] [answered]
Hi,

I'm a bit confused: In the [docs](https://textual.textualize.io/widgets/radioset/#textual.widgets._radio_set.RadioSet.action_toggle_button) I read about a `RadioSet` method `action_toggle_button`. I need it to chose one of the `RadioButtons` programmatically. But it ends with the error message `AttributeError: 'RadioSet' object has no attribute 'action_toggle_button'`. In the minimal example it raises pressing the Button:

```python
from textual.app import App
from textual.widgets import Button, RadioButton, RadioSet

class Main(App):
  
    def compose(self):
        yield Button('Next Choice')
        with RadioSet():
            yield RadioButton('One')
            yield RadioButton('Two')
            yield RadioButton('Three')
    
    def on_button_pressed(self, event):
        self.query_one(RadioSet).action_next_button()
        self.query_one(RadioSet).action_toggle_button()
        # self.query_one(RadioSet).action_toggle()
    
if __name__ == "__main__":
    app = Main()
    app.run()
```

If I replacie `action_toggle_button` with `action_toggle`, then it works.

I'm working with textual version 0.52.1

Did I miss some changes?

By the way: Is there a elegant way to choose one of the `RadioButton` programmatically? It could be the counterpart to `RadioButton.pressed_index` like `RadioButten.set_pressed_index(index)`

Best regards
Ulrich

**Comment by davep** [comment, marked as answer]
You're working with a version of Textual that is older than the documentation. You may first want to update to a more recent version.
